pkg/machinery/config/types/v1alpha1/v1alpha1_types.go

```diff
@@ -1569,6 +1581,13 @@ type APIServerConfig struct {
 	//   examples:
 	//     - value: admissionControlConfigExample
 	AdmissionControlConfig AdmissionPluginConfigList `yaml:"admissionControl,omitempty"`
+	//   description: |
+	//     Configure the API server audit policy.
+	//
+	//     Defaults to the audit policy shown in the example.
+	//   examples:
+	//     - value: auditPolicyConfigDefault
+	AuditPolicyConfig Unstructured `yaml:"auditPolicy,omitempty"`
```

Comment:
we'd need to sill generate the existing default configs in `init` and `worker` files if none is provided,

Reply:
it defaults to existing default policy if not set, so we could print it into the default config... idk, both ways are valid

Reply:
i think it makes more sense to add to the generated config since it's now a fieldd in the machineconfig, less confusion, rather than having a default. Ahh on second thoughts I think it's fine. Just need to reflect in release notes. I'm little conflicted here :sweat_smile:

Reply:
we can do both actually... nothing stops us from doing both: for backwards compat, we inject the default if missing, and also make it explicit in the generated machine config

Reply:
> we can do both actually... nothing stops us from doing both: for backwards compat, we inject the default if missing, and also make it explicit in the generated machine config

sounds like a plan :+1:

Reply:
updated this way!